CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2021-07-26
+| What's new | |
+| :-- |  :-: |
+| Default Headers are no longer opt-in. In the Explorer Settings tab, your Graph Administrators can specify headers are applied to every Explorer request executed by every user in your organization. | ![image](https://user-images.githubusercontent.com/24704789/127037888-2c0c5de2-3ffb-44f1-95c1-ca4888907c73.png) |
+| Also, you can now provide sensitive tokens and information to header values or preflight scripts in Explorer's Environment Variables tab. | ![image](https://user-images.githubusercontent.com/24704789/127039872-c3aebc49-387a-4ac0-b6d5-8eab92cff71c.png) |
```

Comment:
can only graph admins do env variables? If so: 
```suggestion
| Graph Administrators can also now provide sensitive tokens and information to header values or preflight scripts in Explorer's Environment Variables tab. | ![image](https://user-images.githubusercontent.com/24704789/127039872-c3aebc49-387a-4ac0-b6d5-8eab92cff71c.png) |
```

Reply:
Separately: 
```suggestion
| Also, you can now provide sensitive tokens and information to header values or preflight scripts in Explorer's Environment Variables tab next to the Headers and Variables tab. | ![image](https://user-images.githubusercontent.com/24704789/127039872-c3aebc49-387a-4ac0-b6d5-8eab92cff71c.png) |
```

Reply:
Oops didn't mean to commit this. I think anybody can do env vars. Only default headers are limited to graph admins